### PR TITLE
Add replication log details to the exception message when replaying ZooKeeper log fails.

### PR DESCRIPTION
**Pull request by minwoox**
Motivation:
Currently, it is challenging to identify the specific replication log that fails during replay.

Modifications:
- Enhanced exception messages by including details of the replication log that encountered failure.

Result:
- You can now easily identify the failing replication log.

### CODE REVIEW

**Review by jrhee17 — APPROVED** [review]
Thanks for the quick fix! 👍    (I wonder if it's worth adding a check at L738 if the `logMeta.replicaId()` is actually the same since that is the only suspicious point I found)

**Comment by minwoox** [comment]
> I wonder if it's worth adding a check at L738 if the logMeta.replicaId() is actually the same since that is the only suspicious point I found

It's already checked here:
https://github.com/line/centraldogma/blob/cb8734ec9c91e749772078e636dec3930fa79e82/server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java#L1033-L1035

**Review by ikhoon — APPROVED** [review]
Looks good!

File: server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java [hunk 1]
```diff
@@ -746,13 +747,24 @@ private synchronized void replayLogs(long targetRevision) {
                 }
             }
         } catch (Throwable t) {
-            logger.error("Failed to replay a log at revision {}; entering read-only mode", nextRevision, t);
+            if (l != null) {
+                logger.error("Failed to replay a log at revision {}; entering read-only mode. replay log: {}",
+                             nextRevision, l, t);
```
[the remark this answers is not in the document]
Comment (minwoox) [reply server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java]: That's a good point. Applied, thanks!

**Pull request merged**